.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,6 @@
 -   id: nbstripout
     name: nbstripout
     description: 'nbstripout: strip output from Jupyter and IPython notebooks'
-    entry: bash -c 'nbstripout "$@"; git add -u' --
+    entry: bash -c 'nbstripout "$@"' --
```

Comment:
```suggestion
    entry: nbstripout
```

Reply:
Please make this change and I'll merge this PR

Reply:
Thanks @kynan :)